File: source/latest/PTP.Web/ClientApp/src/app/models/journey.model.ts
export class Journey {
  id: number;
  userId: number;
  name: string;
  description: string;
  country: string;
  place: string;
  startDate: Date;
  endDate: Date;
  currency: string;
  status: string;
  amount: number;

  constructor(
    id: number,
    userId: number,
    name: string,
    description: string,
    country: string,
    place: string,
    startDate: Date,
    endDate: Date,
    currency: string,
    status: string,
    amount: number,
  ) {
    this.id = id;
    this.userId = userId;
    this.name = name;
    this.description = description;
    this.country = country;
    this.place = place;
    this.startDate = startDate;
    this.endDate = endDate;
    this.currency = currency;
    this.status = status;
    this.amount = amount;

  }
}

File: source/latest/PTP.BusinessService/Models/JourneyModel.cs
using PTP.DAL.Domains;

namespace PTP.BusinessService.Models;

public class JourneyModel
{
    public virtual int Id { get; set; }
    public virtual UserInfo User { get; set; }
    public virtual string Name { get; set; }
    public virtual string Description { get; set; }
    public virtual Country Country { get; set; }
    public virtual Place Place { get; set; }
    public virtual DateTime StartDate { get; set; }
    public virtual DateTime EndDate { get; set; }
    public virtual long Amount { get; set; }
    public virtual Currency Currency { get; set; }
    public virtual string Status { get; set; }
}